.github/workflows/tests.yml

```diff
@@ -56,6 +56,12 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox installation
+        run: |
+          git clone --depth 1 https://github.com/neuropoly/spinalcordtoolbox
+          cd spinalcordtoolbox
+          yes | ./install_sct
```

Comment:
```suggestion
          yes 2>/dev/null | ./install_sct
```

Let's try the [StackOverflow solution](https://stackoverflow.com/a/20624315/6310633) to see if it's a peculiarity of `yes` throwing an error....

Reply:
I'm also running this locally (without redirecting stderr) on my Mac to see what happens)

Reply:
![image](https://user-images.githubusercontent.com/12765385/101421805-30985580-38c3-11eb-8916-ea75316e26fd.png)
Looks like I made it past the conda installation, so that's already better than the GH Actions build